pages/_document.js

```diff
@@ -1,18 +1,17 @@
+import React from 'react'
 import Document, { Head, Main, NextScript } from 'next/document'
 import { ServerStyleSheet } from 'styled-components'
-import Router from 'next/router'
 
 const DESCRIPTION =
-  `Open-source version control system for Data Science and` +
-  ` Machine Learning projects. Track your data, models, and experiments with` +
-  ` a Git-like tool.`
+  `Open-source version control system for Data Science and Machine Learning ` +
+  `projects. Track your data, models, and experiments with a Git-like tool.`
 const KEYWORDS = `data version control machine learning models management`
 
 const inject = str => (
   <div className="inject" dangerouslySetInnerHTML={{ __html: str }} />
 )
 
-export default class MyDocument extends Document {
+export default class Documentation extends Document {
```

Comment:
this is probably not a good name - sounds like a plural term, while it's actually a singe page

Reply:
I thought about `DocumentationDocument` but it seemed too long. `DocumentationDoc` is confusing since both words start with "doc".

We agree that `MyDocument` is a terrible option right? Seems like it was copy-pasted from a Next tutorial and never updated 😋

Reply:
> We agree that MyDocument is a terrible option right? Seems like it was copy-pasted from a Next tutorial and never updated

🤝 

How about Page, or SitePage - it's a general component for all pages, not only docs as far as I understand.

Reply:
Oh, I almost forget about this one. Ah OK good point. Changed to `Page`.